packages/manager/core/generator/app/conditional-templates/listing/index-api-v6-pagination-step.tsx.hbs

```diff
@@ -3,16 +3,13 @@ import { useTranslation } from 'react-i18next';
 import { {{#if isPCI }} useParams, {{/if}} useNavigate, useLocation } from 'react-router-dom';
 import { useQuery } from '@tanstack/react-query';
 
-import { OsdsLink } from '@ovhcloud/ods-components/react';
-import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
-
+import { OsdsLink, OsdsMessage, OsdsText } from '@ovhcloud/ods-components/react';
+import { ODS_THEME_COLOR_INTENT, ODS_THEME_TYPOGRAPHY_SIZE } from '@ovhcloud/ods-common-theming';
 import {
   Datagrid,
   DataGridTextCell,
-  useDatagridSearchParams,
 } from '@ovhcloud/manager-components';
-
-import { getListingIcebergV6 } from '@/data/api/{{appName}}';
+import useResourcesIcebergV6 from '@ovhcloud/manager-components/src/hooks/datagrid/useIcebergV6';
```

Comment:
Directly import it from the existing import from `manager-components` above.

Reply:
👍